mod/01_core/set/all/fetch.rb

```diff
@@ -54,8 +54,11 @@ def fetch mark, opts={}
     if card.new_card?
       if opts[:new]
         return card.renew(opts) if !clean_cache_opts? opts
+      elsif opts[:skip_virtual]
```

Comment:
>The original logic did just return in this case without testing virtual?

yes, the original logic was `return unless !opts[:skip_virtual] && card.virtual?`